modules/007-registrypackages/images/lsblk/werf.inc.yaml

```diff
@@ -0,0 +1,42 @@
+{{- $version := "2.40.2" }}
+{{- $image_version := $version | replace "." "-" }}
+---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-{{ $image_version }}
+from: {{ $.Images.BASE_SCRATCH }}
+import:
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
+  add: /
+  to: /
+  includePaths:
+  - lsblk
+  - install
+  - uninstall
+  before: setup
+docker:
+  LABEL:
+    distro: all
+    version: all
+    lsblk: {{ $version }}
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
+from: {{ $.Images.BASE_GOLANG_20_ALPINE_DEV }}
+git:
+  - add: /{{ $.ModulePath }}modules/007-{{ $.ModuleName }}/images/{{ $.ImageName }}/scripts
+    to: /
+    stageDependencies:
+      setup:
+      - '**/*'
+shell:
+  install:
+    - apk update && apk add gettext-dev
+  setup:
+    - git clone -b v{{ $version }} --depth 1 {{ $.SOURCE_REPO }}/util-linux/util-linux.git  /src
+    - cd /src
+    - export LDFLAGS="-Wl,-z,now -Wl,-z,relro -static -s" 
+    - export CFLAGS="-fPIC -pie -fstack-protector-all -O2 -D_FORTIFY_SOURCE=2 -static -s"
+    - ./autogen.sh
+    - ./configure --enable-static --enable-static-programs=lsblk --disable-pylibmount --without-python --disable-liblastlog2
```

Comment:
repo util-linux is already used to build fdisk/sfdisk/growpart ( `modules/007-registrypackages/images/growpart/werf.inc.yaml` ), why not add the build and lsblk in there instead of duplicating the build in a separate image?

Reply:
It is possible to merge, but then the name of the package will not reflect its contents.
Also versioning is tied to growpart and util-linux is just a dependency.
Using shared images will make the build more complex and less transparent.